Content.Client/_DV/AACTablet/UI/AACBoundUserInterface.cs

```diff
@@ -20,10 +24,28 @@ protected override void Open()
         _window?.Close();
         _window = this.CreateWindow<AACWindow>();
         _window.PhraseButtonPressed += OnPhraseButtonPressed;
+        _window.Typing += OnTyping;
+        _window.SubmitPressed += OnSubmit;
     }
 
     private void OnPhraseButtonPressed(List<ProtoId<QuickPhrasePrototype>> phraseId)
     {
         SendMessage(new AACTabletSendPhraseMessage(phraseId));
     }
+
+    private void OnTyping()
+    {
+        if (!EntMan.TrySystem(out TypingIndicatorSystem? typingIndicatorSystem))
+            return;
```

Comment:
use UiSystemDependency instead of doing this

Reply:
Is there an additional step to get that to inject with that? AAC's not a UIController so it doesn't appear to ever get the service with that attribute.

Reply:
right use `_typing ??= EntMan.System<TypingIndicatorSystem>();` before using it